.pipelines/templates/build-package-for-ios-cocoapods.yml

```diff
@@ -173,13 +173,13 @@ jobs:
       POD_STAGING_DIR="$(Build.BinariesDirectory)/pod_staging"
       ARTIFACTS_STAGING_DIR="$(Build.ArtifactStagingDirectory)"
       POD_NAME="onnxruntime-extensions-c"
-      POD_ARCHIVE_BASENAME="pod-archive-${POD_NAME}-${ORT_EXTENSIONS_POD_VERSION}.zip"
+      POD_ARCHIVE_BASENAME="onnxruntime_extensions.xcframework.zip"
```

Comment:
nit: slight preference for keeping the version in the zip filename so we know what exactly is in it if someone downloads the artifact directly from a CI build. 

when we rename to include in the nuget package the version info is coming from nuget.